algorithms/kernel/dtrees/gbt/gbt_model_impl.h

```diff
@@ -127,19 +143,34 @@ class GbtDecisionTree: public SerializationIface
             parents[i] = nullptr;
         }
 
+        tree->nNodeSplitFeature.resize(countFeature);
+        tree->CoverFeature.resize(countFeature);
+        tree->GainFeature.resize(countFeature);
+
+        for (size_t i = 0; i < countFeature; ++i)
+        {
+            tree->nNodeSplitFeature[i] = 0;
```

Comment:
use service_memset instead

Reply:
I can't use service_memset now because there is no 'cpu' template parameter in function 'internalTreeToGbtDecisionTree'. Should I add it or just copy the code from service_memset?

Reply:
Seems we can deal without service_memset here.